src/st/class-wpml-pb-integration.php

```diff
@@ -272,15 +282,20 @@ public function migrate_location( $post_id ) {
 			$wpdb = $this->sitepress->get_wpdb();
 			$post = $wpdb->get_row( $wpdb->prepare( "SELECT ID, post_type, post_status, post_content FROM {$wpdb->posts} WHERE ID = %d", $post_id ) );
 			if ( $this->is_post_status_ok( $post ) && $this->is_original_post( $post ) ) {
-				$this->with_strategies( function( IWPML_PB_Strategy $strategy ) use ( $post_id, $post ) {
-					$strategy->migrate_location( $post_id, $post->post_content );
-				} );
+				$this->with_strategies( invoke( 'migrate_location' )->with( $post_id, $post->post_content ) );
 			}
 
 			$this->mark_migrate_location_done( $post_id );
 		}
 	}
 
+	public function register_strings_in_content( $registered, $post_id, $content ) {
```

Comment:
@brucepearson, I can see that you did not add a PHPDoc everywhere, it's missing in several places. Could you fix it please?

Reply:
@strategio I added.